ansible_collections/ds/ansible_ds/tests/README.md

```diff
@@ -0,0 +1,16 @@
+389 Directory Server Ansible collection tests
+==========================
+
+This directory contains the test framework
+
+Note: tests are either:
+    - test_xxxx methods within python file or 
+    - test_xxxx.yml playbooks
+
+They can be run:
+    - directly from pytest. (Usefull when wanting to run a single test: i.e pytest $PWD -k test_gather_dsinst_info.yml)
```

Comment:
```
pytest $PWD -k test_gather_dsinst_info.yml
```

it gives `collected 1 item / 1 deselected / 0 selected` error.
I think it because only the `.py` test is found

Reply:
should be fixed in latest commit